### DISCUSSION TITLE
1.5 roles split between repositories

### DISCUSSION THREAD

**Discussion by oxedions** [bluebanquise/bluebanquise#520] [General]
Hello 😊 

With start of 1.5, first step before KISS pass over Core is to dispatch roles between repositories.

Here is a proposal, with suggested owners (< XX) and sometime comments:

----
* CORE
    * access_control
    * bluebanquise # should be optional (but kept), since requirements should be added in the rpm spec of main bluebanquise
    * dhcp_server
    * dns_client
    * dns_server
    * firewall
    * hosts_file
    * log_client
    * log_server
    * nfs_client
    * nfs_server
    * nic # To be replaced by nic_nmcli role
    * pxe_stack
    * repositories_client
    * repositories_server
    * set_hostname
    * ssh_master
    * ssh_slave
    * time
    * diskless
    * kernel_config
    * lvm
    * root_password

----
* COMMUNITY
    * conman < @hmeAtos ?
    * display_tuning < @oxedions 
    * advanced_dhcp_server < @hmeAtos
    * advanced_dns_server < @oxedions 
    * clustershell < @oxedions 
    * grafana < @oxedions 
    * lmod < @oxedions 
    * nhc < @oxedions 
    * ofed < @oxedions 
    * ofed_sm < @oxedions 
    * openldap_client
    * openldap_server
    * powerman
    * prometheus_client < @oxedions 
    * prometheus_server < @oxedions 
    * report < @oxedions 
    * singularity < @strus38 ?
    * slurm < @oxedions 
    * sssd
    * users_basic < @oxedions 

----

Remember that Community roles can have their own release cycle, so fast if needed. This is one of the main purposes of this change.

**Comment by btravouillon** [comment]
I'm sorry I won't be able to maintain _advanced_dns_server_ as a community role.

**Reply by oxedions** [reply]
Ok, I understand. I will take care of it then.

**Comment by oxedions** [comment]
Multiple users are not big fans of this repository split. @johnnykeats Can we discuss about this ?
